common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -97,6 +97,15 @@ class ProctoringFields(object):
         scope=Scope.settings,
     )
 
+    hide_after_due = Boolean(
+        display_name=_("Hide Exam Results After Due Date"),
+        help=_(
+            "This setting overrides the default behavior of showing exam results after the due date has passed."
```

Comment:
@efischer19 Are we using "timed exams" to include proctored exams (which are always timed)?
If yes, should this note read "Currently supported only for timed and proctored exams" for clarity? 
Also would recommend getting rid of initial caps for "Timed Exams".